Lib/test/test_fstring.py

```diff
@@ -321,8 +315,8 @@ def test_ast_line_numbers_multiline_fstring(self):
         self.assertEqual(binop.lineno, 4)
         self.assertEqual(binop.left.lineno, 4)
         self.assertEqual(binop.right.lineno, 6)
-        self.assertEqual(binop.col_offset, 3)
-        self.assertEqual(binop.left.col_offset, 3)
+        self.assertEqual(binop.col_offset, 4)
+        self.assertEqual(binop.left.col_offset, 4)
```

Comment:
I suspect this was incorrect before (being adjusted by `-1` due to the multiline string).

For instance, here's the ast dump of (using [astpretty](https://github.com/asottile/astpretty))

```python
x = 1 + 1
```

```
$ python ~/workspace/astpretty/astpretty.py t.py 
Module(
    body=[
        Assign(
            lineno=1,
            col_offset=0,
            targets=[Name(lineno=1, col_offset=0, id='x', ctx=Store())],
            value=BinOp(
                lineno=1,
                col_offset=4,
                left=Num(lineno=1, col_offset=4, n=1),
                op=Add(),
                right=Num(lineno=1, col_offset=8, n=1),
            ),
        ),
    ],
)
```

You'll notice that the `BinOp` and its `left` (`Num`) have the same `col_offset`